src/components/seo.js

```diff
@@ -72,7 +77,14 @@ const SEO = ({
           name: 'twitter:description',
           content: metaDescription,
         },
-      ].concat(meta)}
+      ].concat(
+        metaKeywords && metaKeywords.length > 0
```

Comment:
We don't need this check here because of line 31,`metaKeywords` won't be undefined and will always have length > 0